src/utils/etherspot.js

```diff
@@ -284,3 +285,37 @@ export const getChainTokenListName = (chain: Chain): ?string => {
 
   return null;
 };
+
+/**
+ * How use this function
```

Comment:
If you're going to use doc blocks, please remember to conform to their standards. We need a `@name` and `@description` here